salt/modules/lxc.py

```diff
@@ -2508,7 +2591,11 @@ def bootstrap(name,
     # default set here as we cannot set them
     # in def as it can come from a chain of procedures.
     if bootstrap_args:
-        bootstrap_args = '{0} -c {{0}}'.format(bootstrap_args)
+        # custom bootstrap args can be totally customized, and user could
```

Comment:
I think the best solution would be for the format call to specify the config dir as a kwarg since it would remove the need to check for `{0}`...